### DISCUSSION TITLE
is there any slowly code on the move_and_slice(or animation_mixer)

### DISCUSSION THREAD

**Discussion by yythlj** [godotengine/godot-proposals#9548] [3D]
### Describe the project you are working on

I test my game on server mode(set headless and dummy).
1.if there is 120 characterbody3d call move_and_slide on every physice_process，there fps become 1 or 2.use move_and_collide behave more well.
2.if only call the animation_player play some bones animation, fps also slowly . 3 or 4 fps one second.through I hide the mesh of the skele.and already set headless mode.
but if I close the animation code only keep the logic code to hold every delta cal, the fps behave good to 140

now I try to make a memory data to ready every delta bones pos and rotation instead of play the animation.

It seem has some bug.
if there is one room only.I can have more than 100 fps 

### Describe the problem or limitation you are having in your project
every room use a window , set its owne world3d
20 room. 6 mixamo bones player one room.hide the mesh.play walk animation randomly.fps slow down

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

a useful way to make animation for headless mode(just postion and roration info need to set, no physic and no display)
.high fps

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

a useful way to make animation for headless mode(just postion and roration info need to set, no physic and no display)
.high fps

### If this enhancement will not be used often, can it be worked around with a few lines of script?

N/A

### Is there a reason why this should be core and not an add-on in the asset library?

N/A

**Comment by AThousandShips** [comment]
Converted to an issue as this:
* Wasn't filled in properly
* Isn't a proposal but a question

**Reply by yythlj** [reply]
my thinking is
other function like advance, but less time call.

like only push the new frame and the delta, so some status change(like active, transition request)
and option to select if the position and rotation also change

Its a way for headless server mode to advance animation in a lower times and less frame use
